### DISCUSSION TITLE
Aliasing a list of ingredients

### DISCUSSION THREAD

**Discussion by rasulkireev** [cooklang/spec#41] [Ideas]
I've looked through issues and discussions, and am pretty sure no one raised this question.

I'm thinking that it would be good/convenient to have a way to group ingredients and reference them in the recipe. For example, let's say you are cooking something with many herbs and spices. And the recipe required you to:
1. toast your spices
2. add spices to broth
3. remove spices from broth

Right now, it would look something like this:

```cooklang
Add @juniper berries{4}, @cardamom pods{3}, @bay leaves{2}, ...., .... and ....

... some other steps...

Then add your toasted spices to the broth. After 20 minutes, remove spices from the broth.
```

And after writing this down, this seems fine actually, because ingredients only need to be listed once. However, for the sake of the argument let's say that we could group ingredients together like so:

```cooklang
Add spices(@juniper berries{4}, @cardamom pods{3}, @bay leaves{2}, ...., ...., ....)

... some other steps...

Then add your toasted @spices to the broth. After 20 minutes, remove @spices from the broth.
```

1. This would allow for nice grouping of ingredients in the header.
2. It could hypothetically help with solving https://github.com/cooklang/spec/issues/36

**Comment by gvlx** [comment]
linking with #68
